returnn/rasr_training.py

```diff
@@ -232,9 +234,22 @@ def create_config(
         return config, post_config
 
     @classmethod
-    def create_dataset_config(cls, train_crp, partition_epochs):
+    def create_dataset_config(cls, train_crp, returnn_config, partition_epochs):
```

Comment:
do we have the situation, where `train_crp` and `dev_crp` rely on different RASR binaries? So no problem taking RASR for both train/dev from `train_crp` right?

Reply:
I cannot imagine a situation where you'd need to use different RASR binaries for train and dev.